mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt

```diff
@@ -46,4 +47,11 @@ abstract class ViewModelTest {
      * It allows to work with the live data.
      */
     @get:Rule val instantTaskExecutorRule = InstantTaskExecutorRule()
+
+    /**
+     * Swap coroutine dispatcher with the one that is executed immediately.
+     * It allows to work with the coroutines.
+     */
+    @ExperimentalCoroutinesApi
+    @get:Rule var coroutineScopeRule = CoroutineScopeRule()
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/854e22184fc68db6b369995b1ab07bd23e0a7a16/mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt#L57"></span>The file ViewModelTest.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>